package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@debitoor/eslint-config-debitoor",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
1. I think the ecma version bump might be considered a major (breaking) change
2. use [npm version](https://docs.npmjs.com/cli/v8/commands/npm-version) to bump packages

Reply:
good point )